dvc/commands/dag.py

```diff
@@ -198,6 +464,30 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Print output files instead of stages.",
     )
+    dag_parser.add_argument(
+        "--direction",
+        choices=["LR", "TD"],
+        default="TD",
+        help=(
+            "Direction of the rendered mermaid DAG. "
+            "Can either be 'LR' for left-to-right or 'TD' for top-down'."
+        ),
+    )
```

Comment:
I would prefer to be opinionated on the representation. 
It is hard to draw a boundary between what can be customized and what not.
In this case, I think it is not worth having an option because users could just:

 `dvc dag --md | sed 's/flowchart TD/flowchart LR/g'`

Reply:
True, it's a simple thing to change. Probably shouldn't bloat the options of this CLI.

Just some real life observation as in our pipeline we have a rather shallow pipeline (like 5-6 stages deep) but a large breadth (training data comes in as 15+ archives, as it is too large to handle otherwise & training and deployment stages are adapted for many different target architectures). So our DAGs look much prettier in left-to-right ;)

Reply:
> So our DAGs look much prettier in left-to-right ;)

I have nothing against making LR the default, though 😄